build.sbt

```diff
@@ -297,6 +297,13 @@ lazy val persistenceTestkit = akkaModule("akka-persistence-testkit")
   .settings(AutomaticModuleName.settings("akka.persistence.testkit"))
   .disablePlugins(MimaPlugin)
 
+lazy val persistenceTypedTests = akkaModule("akka-persistence-typed-tests")
```

Comment:
Needed to be a new module to avoid cyclic dependency between testkit and the prod module